.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 node_modules
 npm-debug.log
+.idea/
```

Comment:
I added some more information to the import-js issue: https://github.com/trotzig/import-js/issues/102#issuecomment-173168629
And I'm happy to help with whatever you need.